src/Arriba/Arriba.Server/Controllers/ArribaController.cs

```diff
@@ -0,0 +1,159 @@
+﻿using Arriba.Communication.Server.Application;
+using Arriba.Model;
+using Arriba.Model.Column;
+using Arriba.Model.Query;
+using Arriba.Model.Security;
+using Arriba.Types;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Net;
+using System.Runtime.CompilerServices;
+
+namespace Arriba.Server.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ArribaController : ControllerBase
+    {
+        private readonly IArribaManagementService _arribaManagement;
+        private readonly IArribaServerConfiguration _arribaServerConfiguration;
+
+        public ArribaController(IArribaManagementService arribaManagement, IArribaServerConfiguration arribaServerConfiguration)
```

Comment:
Feels like IArribaServerConfiguration will go away if the exception handling is moved to a middleware